ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/WorkloadTableToolbar.tsx

```diff
@@ -20,30 +22,37 @@ type WorkloadTableToolbarProps = {
     defaultFilters?: DefaultFilters;
     supportedResourceFilters?: FilterAutocompleteSelectProps['supportedResourceFilters'];
     autocompleteSearchContext?: FilterAutocompleteSelectProps['autocompleteSearchContext'];
+    onFilterChange?: (searchFilter: SearchFilter) => void;
 };
 
 function WorkloadTableToolbar({
     defaultFilters = emptyDefaultFilters,
     supportedResourceFilters,
     autocompleteSearchContext,
+    onFilterChange = noop,
```

Comment:
oh what's `noop` do here and how is it different than `undefined` or `null` ?

Reply:
`noop` is essentially just `() => {}`, and I used it in this case so that the default value for the function isn't different on each render. This felt a little cleaner than making the optional prop have no default and having to do `if (onFilterChange)` checks when calling the function.